hbase-server/src/main/java/org/apache/hadoop/hbase/wal/BoundedGroupingStrategy.java

```diff
@@ -72,7 +72,7 @@ public void init(Configuration config, String providerId) {
     int regionGroupNumber = config.getInt(NUM_REGION_GROUPS, DEFAULT_NUM_REGION_GROUPS);
     groupNames = new String[regionGroupNumber];
     for (int i = 0; i < regionGroupNumber; i++) {
-      groupNames[i] = providerId + GROUP_NAME_DELIMITER + "regiongroup-" + i;
```

Comment:
GROUP_NAME_DELIMITER is unused from now on. Can it be removed? It is in IA.Private class.